scripts/e2e.surge.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# --------------------------------------------------------------------
+# Publishes a small test site that uses the packages/web3/dist/min
+# --------------------------------------------------------------------
+
+if [ -z "$CI" ]; then
+
+  echo "======================================================================"
+  echo "This script publishes a test site with surge.sh. Only run in CI."
+  echo "======================================================================"
+
+  exit 1
+
+fi
+
+echo "======================================================================"
+echo "Publishing test site with surge.sh. See link below.                   "
+echo "======================================================================"
+
+cp packages/web3/dist/web3.min.js scripts/html
+cd scripts
+(echo "" && exit && cat ) | npx surge html
+
+# Might be running locally in development
+rm html/web3.min.js
+
+# Script doesn't always work...
```

Comment:
Is it possible to fix the described error from the PR desc? could you add some more details about it? 🤔

Reply:
Added in https://github.com/ethereum/web3.js/pull/3261/commits/331b01176fc0c4f01bd066f172d0b303f84b8ab9

Reply:
Just curious about, does this mean the ``surge`` error which appears sometimes isn't fixable? :)

Reply:
I think it's something about surge - below is from a  [surge guide](https://sunlightmedia.org/using-surge-for-deploying-static-sites/) and the error described is what I've seen, using random names. But if you want to make an account for yourself since you're the Web3 maintainer, it would probably work to publish to same site each time. 

It's [surge.sh](https://surge.sh/help/) and free.

> Surge will automatically provide a domain name using random words. You can also create any custom domain if it is not already taken. Enter your chosen domain name, then hit enter.

> (Note: If the custom domain name you input is already taken, you’ll see an error message.

> Aborted - you do not have permission to publish to [<custom-domain.surge.sh>]

Reply:
@nivida I've just pushed a change to make this always publish to sudden-playground.surge.sh since that name works. If you need to change this, it super easy - there are a couple tokens to set in CI.